devops/ansible/arango-ssl.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+cat /etc/letsencrypt/live/SERVER_NAME/fullchain.pem /etc/letsencrypt/live/SERVER_NAME/privkey.pem > /home/ubuntu/server.pem
+chmod 644 /home/ubuntu/server.pem
+
+systemctl restart arangodb3
```

Comment:
Is this script something that has to run on renewal? If not, these steps should be converted to Ansible playbook steps. If yes, maybe a comment block in the script explaining why these are not done via Ansible would be good.

Reply:
Yeah, this runs on a successful renewal hook. Thus when the cert renews, this is triggered

Reply:
Fixed in 196b0df

Reply:
This is fine for now, but it occurs to me that you can absolutely just put these steps in an independent playbook and invoke that on renewal.